service/deployer/eventer/github/spec.go

```diff
@@ -45,5 +45,5 @@ var (
 	// successState is the state for successful Deployment Status states.
 	successState deploymentStatusState = "success"
 	// failedState is the state for failed Deployment Status states.
-	failedState deploymentStatusState = "failed"
+	failedState deploymentStatusState = "failure"
```

Comment:
You know what I am thinking about the variable name now, don't you? :D You do. Yeah boy.